libs/device-protocol/feature/src/services/serial-port.service.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import SerialPort from "serialport"
+
+export interface ISerialPortService {
+  list(): Promise<SerialPort.PortInfo[]>
+}
+
+export class SerialPortService implements ISerialPortService {
+  list(): Promise<SerialPort.PortInfo[]> {
+    return SerialPort.list()
```

Comment:
it probably should use mock devices list when mock service is enabled

Reply:
https://github.com/mudita/mudita-center/pull/1950/files#diff-73c889eb7b000757a16fa2a10d50a2826b9b0253fe584bc0b47f1e0f4b0b32c9R157